credit-check-service/src/main/java/com/senacor/innolab/graalvm/integration/neo4j/model/CreditDetails.java

```diff
@@ -6,11 +6,18 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
 @Data
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
 @RegisterForReflection
 public class CreditDetails {
     private String Id;
+    private BigDecimal amount;
+    private LocalDate startTime;
+    private LocalDate endTime;
```

Comment:
I'd rename `startTime` and `endTime` to `~Date` respectively. Credits are more about dates than times